static/app/components/group/groupSummary.tsx

```diff
@@ -125,91 +92,133 @@ export function GroupSummary({groupId, groupCategory}: GroupSummaryProps) {
                   }}
                 />
-              </ImpactContent>
-            </Content>
+                <ImpactContent>
+                  <SummaryContent
+                    dangerouslySetInnerHTML={{
+                      __html: marked(`**Impact:** ${data.impact}`),
+                    }}
+                  />
+                </ImpactContent>
+              </Content>
+            )}
+          </div>
+          {openForm && !isPending && (
+            <ButtonContainer>
+              <Button
+                onClick={() => {
+                  openForm({
+                    messagePlaceholder: t(
+                      'How can we make this issue summary more useful?'
+                    ),
+                    tags: {
+                      ['feedback.source']: 'issue_details_ai_issue_summary',
+                      ['feedback.owner']: 'ml-ai',
+                    },
+                  });
+                }}
+                size="xs"
+                icon={<IconMegaphone />}
+              >
+                Give Feedback
+              </Button>
+              <GroupSummaryFeatureBadge />
+            </ButtonContainer>
           )}
-        </div>
-        {openForm && !isPending && (
-          <ButtonContainer>
-            <Button
-              onClick={() => {
-                openForm({
-                  messagePlaceholder: t(
-                    'How can we make this issue summary more useful?'
-                  ),
-                  tags: {
-                    ['feedback.source']: 'issue_details_ai_issue_summary',
-                    ['feedback.owner']: 'ml-ai',
-                  },
-                });
-              }}
-              size="xs"
-              icon={<IconMegaphone />}
-            >
-              Give Feedback
-            </Button>
-          </ButtonContainer>
-        )}
-      </Wrapper>
-    </SidebarSection.Wrap>
+        </Body>
+      )}
+    </Wrapper>
   );
 }
 
+const Body = styled('div')`
+  padding-left: ${space(3)};
+  padding-right: ${space(3)};
+  padding-top: ${space(1)};
+  padding-bottom: ${space(1.5)};
+`;
+
+const Headline = styled('p')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: ${space(0.5)};
+  flex-shrink: 0;
+  margin-top: ${space(3)};
+`;
+
+const SummaryPreview = styled('p')`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  flex-grow: 1;
+  color: ${p => p.theme.subText};
+  margin-top: ${space(3)};
+`;
+
 const Wrapper = styled(Panel)`
-  display: flex;
-  flex-direction: column;
   margin-bottom: 0;
-  background: linear-gradient(
-    269.35deg,
-    ${p => p.theme.backgroundTertiary} 0.32%,
-    rgba(245, 243, 247, 0) 99.69%
-  );
-  padding: ${space(1.5)} ${space(2)};
+  padding: ${space(0.5)};
 `;
 
 const StyledTitleRow = styled('div')`
   display: flex;
   align-items: center;
   justify-content: space-between;
+  height: ${space(2)};
```

Comment:
prefer min height to height
```suggestion
  min-height: ${space(2)};
```